src/Traits/Auth/AuthenticatesRequests.php

```diff
@@ -32,17 +35,26 @@ protected function defaultAuth(): ?Authenticator
      */
     public function getAuthenticator(): ?Authenticator
     {
-        return $this->authenticator ?? $this->defaultAuth();
+        if (isset($this->authenticator)) {
+            return $this->authenticator;
+        }
+
+        if ($defaultAuth = $this->defaultAuth()) {
+            $this->authenticate($defaultAuth);
+        }
+
+        return $this->authenticator;
```

Comment:
Could you perhaps shorten this code to:

```php
return $this->authenticator ?? $this->authenticate($this->defaultAuth()) ?? $this->authenticator;
```

Reply:
And isn't `$this->authenticator` duplicated? I can see it the first and the last?

Reply:
I've decided to revert to the old code and just accept a `MultiAuthenticator` instance instead of an array or an authenticator - keeps it simpler!